tools/topology/sof-apl-dmic.m4

```diff
@@ -23,7 +23,13 @@ define(DMIC_PDM_CONFIG, ifelse(CHANNELS, `4', ``FOUR_CH_PDM0_PDM1'',
 #
 # Define the pipelines
 #
+ifelse(EQ, `enabled',
+`
+# PCM6 <---- EQ IIR <----- DMIC6 (DMIC01)
+    define(PIPE_VOLUME_CAPTURE, `sof/pipe-eq-capture.m4')',
```

Comment:
Could it be instead PIPE_PROCESS_CAPTURE since it will be replaced by other than volume frequently. Please see also my more general suggestion in conversation page for this PR.